app/models/preserved_object.rb

```diff
@@ -22,8 +22,24 @@ class PreservedObject < ApplicationRecord
   # Log an error message.
   # Calls ReplicatedFileCheckJob
   # This builds off of #917
-  def check_endpoints!
+  def check_zip_endpoints!
     # FIXME: STUB
     # Ticket: 920
   end
+
+  # given a version, create any PreservedCopy records for that version which don't yet exist for zip
```

Comment:
ZippedMoabVersion instead of PreservedCopy?  Could just mention this in the PreservedCopy rename ticket instead.

Reply:
put a note in #976

Reply:
actually, my guess is this is an errant chunk of code that got put back in (erroneously) upon rebase.  i think this method should disappear from PO entirely, as that functionality now lives here:  https://github.com/sul-dlss/preservation_catalog/blob/1d32d8f88ab4d89d453d1153003322f48ba4350c/app/models/preserved_copy.rb#L73-L88

Reply:
whoops! the rebase was messy...